### DISCUSSION TITLE
Random locale

### DISCUSSION THREAD

**Discussion by ahaberbosch** [faker-js/faker#2285] [Q&A]
There was a way to select a random locale to be used with setLocale, but that support was deprecated. Is there still a way to do this? I'd like to randomize getting foreign addresses but there doesn't seem to be a way to do this anymore. Maybe with combination of lodash sample method?

**Comment by matthewmayer** [comment]
all the precompiled Faker instances are acessible in an object called allFakers
```js
const {allFakers, faker} = require("@faker-js/faker")
const randomFaker = faker.helpers.objectValue(allFakers)
randomFaker.location.city()
```

**Reply by Shinigami92** [reply]
I would recommend to use modern syntax: `import { allFakers, faker } from "@faker-js/faker";`, but yes, that is a solution.

And also no bundle-size problem for you (@ahaberbosch), because you seemed to try to get all data before the deprecation anyways with also loading all faker data. But keep in mind that this are around 5mb of Data and we not recommend to use that in production.
